gui-wm/sway/sway-1.7-r1.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="i3-compatible Wayland window manager"
+HOMEPAGE="https://swaywm.org"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/swaywm/${PN}.git"
+else
+	MY_PV=${PV/_rc/-rc}
+	SRC_URI="https://github.com/swaywm/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64"
```

Comment:
Is there a reason for the dropped keywords?  We shouldn't drop keywords if we can help it.

Reply:
Well, the new dependencies are not (yet) keyworded for these archs.
For example: `gui-apps/grim` is only `~amd64 ~arm64` keyworded. `gui-apps/slurp` is only keyworded for `~amd64 ~arm64 ~x86`

I guess a better approch would be to use.mask `grimshot` for these archs for the meantime?

Reply:
nah, better would be to get them keyworded so we can add them here. (just did so)

Reply:
well that's of course even better :)
We still need the keywords for `gui-apps/slurp`, then i should be able to restore all keywords (until then, repoman won't be happy)